back/spec/acceptance/ideas_spec.rb

```diff
@@ -552,8 +554,9 @@
     end
 
     get 'web_api/v1/ideas/:id' do
-      let(:idea) { create(:idea) }
-      let!(:baskets) { create_list(:basket, 2, ideas: [idea]) }
+      let(:project) { create(:continuous_budgeting_project) }
```

Comment:
Because it didn't make sense to have baskets on a project without a voting phase. Given that the test isn't specific to any particular scenario, I thought I'd at least make it match a real situation